data/json/vehicle_parts.json

```diff
@@ -3185,6 +3203,44 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "wood box",
+        "name" : "wood box",
+        "symbol" : "o",
+        "color" : "brown",
+        "broken_symbol" : "#",
+        "broken_color" : "brown",
+        "damage_modifier" : 60,
+        "durability" : 70,
+        "size" : 300,
```

Comment:
Wooden frame is less durable than steel frame, so it should be smaller than the "box" part made of steel (that has 275 volume). 200 would be plenty, even 100 would be fine. It can be made very cheaply and with no skills, so it shouldn't be better than expensive parts.